src/main/java/org/example/app/service/TwitService.java

```diff
@@ -0,0 +1,68 @@
+package org.example.app.service;
+
+import lombok.AllArgsConstructor;
+import lombok.SneakyThrows;
+import org.example.app.repository.TwitRepository;
+import org.example.app.elasticclient.TwitElasticsearchClient;
+import org.example.app.entity.Twit;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+@AllArgsConstructor
+public class TwitService {
+
+    @Autowired
```

Comment:
Since you're using all args constructor annotation field injection is not required I believe

Reply:
removed all args constructor annotation